lib/widgets/login.dart

```diff
@@ -648,18 +649,22 @@ class _UsernamePasswordFormState extends State<_UsernamePasswordForm> {
           selectedIcon: const Icon(Icons.visibility_off),
         )));
 
-    return Form(
-      // TODO(#110) Try to highlight CZO / Zulip Cloud realms in autofill
-      child: AutofillGroup(
-        child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
-          usernameField,
-          const SizedBox(height: 8),
-          passwordField,
-          const SizedBox(height: 8),
-          ElevatedButton(
-            onPressed: widget.loginPageState._inProgress ? null : _submit,
-            child: Text(zulipLocalizations.loginFormSubmitLabel)),
-        ])));
+    return Semantics(
```

Comment:
How did you decide this `Semantics` was necessary? From a quick skim of the implementation of the upstream `Form` widget, it looks like this is already taken care of:

```dart
    return Semantics(
      container: true,
      explicitChildNodes: true,
      role: SemanticsRole.form,
      child: form,
    );
```